### ISSUE TITLE
Add a new option for `scss/dollar-variable-empty-line-before` after functions

### ISSUE THREAD

**Issue by AlexWayfer** [stylelint-scss/stylelint-scss#1096]
Hello.

I have such code:

```scss
@use "sass:color";

@use "../lib/inputs-with-types";

@use "../colors";
@use "../sizes";

$form-vertical-label-margin: 0.8em;
$form-vertical-input-margin: 0.4em;

@mixin styles {
  /* ... */
}
```

And there are such offenses:

```
assets/styles/components/_forms.scss
  8:1  ✖  Unexpected empty line before $-variable  scss/dollar-variable-empty-line-before
```

The suggestion is to have something like `after-functions` option for `ignore` configuration.

**Comment by jack-samo10000** [comment]
Hello, we tried to solve the issue.

This is what we did:

This change adds a new 'after-use' option to the 'ignore' configuration for the dollar-variable-empty-line-before rule. This allows users to ignore $-variables that come immediately after @use statements, addressing the issue described in the GitHub issue.

You can review changes in this commit: https://github.com/jack-samo10000/stylelint-scss-stylelint-scss-1096/commit/162ce298dbfa4affcdffc7f09c449f2a28bf3864.


> [!CAUTION]
> Disclaimer: The concept of solution was created by AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---

Latta AI seeks to solve problems in open source projects as part of its mission to support developers around the world. Learn more about our mission at https://latta.ai/ourmission . If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.
